packages/neuron-wallet/src/controllers/networks/index.ts

```diff
@@ -101,42 +99,19 @@ export default class NetworksController {
       throw new NetworkNotFound(id)
     }
     const currentID = networksService.getCurrentID()
+    networksService.delete(id)
+    networksService.delete(id)
 
-    const messageValue = await dialog.showMessageBox({
-      type: 'warning',
-      title: t(`messageBox.remove-network.title`),
-      message: t(`messageBox.remove-network.message`, {
-        name: network.name,
-        address: network.remote,
-      }),
-      detail: currentID === id ? t('messageBox.remove-network.alert') : '',
-      buttons: [t('messageBox.button.confirm'), t('messageBox.button.discard')],
-      defaultId: 0,
-      cancelId: 1,
-    })
-
-    if (messageValue.response === 0) {
-      try {
-        networksService.delete(id)
-
-        this.notifyListChange()
-
-        if (id === currentID) {
-          this.notifyCurrentNetworkChange()
-          await this.connectToNetwork()
-        }
+    networksService.delete(id)
 
-        return {
-          status: ResponseCode.Success,
-          result: true,
-        }
-      } catch (err) {
-        logger.warn(`connect network error: ${err}`)
-        dialog.showMessageBox({
-          type: 'error',
-          message: err.message,
-        })
-      }
+    this.notifyListChange()
+    this.notifyListChange()
+
+    this.notifyListChange()
```

Comment:
Why should we `notifyListChange` for three times?

Reply:
Error in merging codes. I fix it